submissions/AnthoniaAdollo/aboutme.txt

```diff
@@ -0,0 +1,4 @@
+Hello World,
```

Comment:
@AAOYV2020 The aboutme text file should be in the aboutMe folder you have created. Also, the name convention for your name and the name of the text file do not follow the required naming convention as in `camelCase`. Well done on coming this far.

Reply:
Thank you for the feedback, Tolu. I have updated the naming convention on the 'Name' folder to camelCase. I apologise for the error. As for the text file, the one for the pull request was actually inside the 'aboutMe' file, as directed. the one outside was created while I was following folashade's tutorial. I will take that out, too